.flowconfig

```diff
@@ -20,6 +25,9 @@
 module.name_mapper='^@keystone\/\(.+\)$' -> '<PROJECT_ROOT>/packages/\1'
 module.name_mapper='.*FIELD_TYPES' -> '<PROJECT_ROOT>/packages/field-views-loader/field-types'
 module.name_mapper='^@arch-ui\/\([^\/]+\)$' -> '@arch-ui/\1/src/index.js'
-server.max_workers=1
+server.max_workers=4
+suppress_comment= \\(.\\|\n\\)*\\$flow:
```

Comment:
Doing this means that consumers would also have to have this suppression comment in their config which we should definitely avoid.

Reply:
there is `$FlowFixMe`, but there are cases where errors don't have to be fixed, like in mocked functions, just a descriptive comment is needed
should I use `$FlowFixMe` instead?

Reply:
Yeah, using $FlowFixMe would be best

Reply:
done